src/android/OpenTokAndroidPlugin.java

```diff
@@ -561,6 +562,14 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
         streamHasAudio = new HashMap<String, Boolean>();
         streamHasVideo = new HashMap<String, Boolean>();
         streamVideoDimensions = new HashMap<String, JSONObject>();
+        String deviceName = Settings.Global.getString(cordova.getContext().getContentResolver(), "device_name");
+        /* DEV-11766 (epic DEV-11304) : Setting custom audio driver for A7 lite devices to enhance volume*/
```

Comment:
Querying the device name and only applying custom audio driver to A7 Lite device